packages/dbs-ai/client/views/app/tabbar/smarti.js

```diff
@@ -38,13 +38,30 @@ Template.dbsAI_smarti.onRendered(function() {
 			const WEBSOCKET_URL =
 				`ws${ RocketChat.settings.get('Site_Url').substring(4) }websocket/`;
 
+			let customSuffix = RocketChat.settings.get('Assistify_AI_DBSearch_Suffix') || '';
+			customSuffix = customSuffix.replace(/\r\n|\r|\n/g, '');
 
-			self.smarti = new window.SmartiWidget(self.find('.external-message'), {
+			const smartiOptions = {
 				socketEndpoint: WEBSOCKET_URL,
 				smartiEndpoint: DBS_AI_Redlink_URL,
 				channel: self.data.rid,
-				inputCssSelector: '.autogrow-shadow'
-			});
+				inputCssSelector: '.autogrow-shadow',
+				widget:{
```

Comment:
@ruKurz the css-selector is also an option for the widget - actually everything in the options is related to the widget. Thus, I'd not introduce a widget-property.